molecule/command/init/role.py

```diff
@@ -72,12 +75,16 @@ def execute(self):
             )
             util.sysexit_with_message(msg)
 
-        template_directory = ''
-        if 'template' in self._command_args.keys():
-            template_directory = self._command_args['template']
-        else:
-            template_directory = 'role'
-        self._process_templates(template_directory, self._command_args, role_directory)
+        try:
+            cmd = ["ansible-galaxy", "init", "-v", "--offline", role_name]
```

Comment:
* Why use `-v` without checking if the user has passed `--debug`?
* We should catch a [CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) instead?
* Should we also `--ignore-certs` to avoid annoying failures?

Reply:
Even verbose mode is not very verbose on Galaxy. Lets keep it like this because we do not have access at config option. We can make it configurable when we have a proper CfgMgr.

I see no reason to touch --ignore-certs option, I am not keen to hardcode insecure defaults. Better to see a bug raised and we can see what we can do, even if I consider that such option should be controllable using environment or ansible.cfg instead of cli.